utility/src/main/java/org/oppia/util/caching/AssetRepository.kt

```diff
@@ -34,19 +37,22 @@ class AssetRepository @Inject constructor(private val context: Context, private
     }
   }
 
-  /** Ensures the contents corresponding to the specified asset are available for quick retrieval. */
+  /** Ensures the contents corresponding to the specified asset are available for
```

Comment:
Eek, this isn't reformatting correctly. Did you do it manually or did ktlint? It should be:

```kotlin
/**
 * Start comment...
 */
```

Is this a ktlint bug?

Ditto elsewhere in this PR.

Reply:
Ktlint isn't able to handle comments unfortunately, it does code reformatting but Android Studio and Ktlint don't do comments. I have just done the comments manually, I will update this